resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -164,68 +155,72 @@ Python 3.7.7
+There is a community-driven effort called the [conda-forge](https://conda-forge.org/),
+which provides more up to date packages
+To enable us to access the most up to date version of the Python packages we are going to use,
+we will add the more up to date  channel,
+To add the conda-forge channel by typing the following in the terminal:
 
 ```
 conda config --add channels conda-forge
 
 ```
 
-Thus, to install other pacakges individually, we can now use the following: conda install <package-name>. We will install the key packages needed for the start of our program now:
+To install packages individually, we can now use the following command: `conda install <package-name>`. Let's install the key packages needed for the start of our program:
 
 ```
-conda install --yes \
+conda install \
  jupyterlab=2.* \
  numpy=1.* \
  pandas=1.* \
  flake8=3.* \
  black=19.*
 ```
 
+`conda` will show you the packages that will be downloaded,
+and you can press enter to proceed with the installation.
+
 > Note: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
 
-## Jupyter extensions
-We will be using several Jupyter extensions that help us use Juypter notebooks more smoothly with Git & GitHub. To install them, paste the following in the terminal below:
+## JupyterLab extensions
+
+We will be using a couple of JupyterLab git extensions to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
 
 ```
-conda install --yes nodejs=10.*
+conda install nodejs=10.*
 pip install --upgrade jupyterlab-git
-conda install --yes jupytext=1.*
+conda install jupytext=1.*
 jupyter lab build
 ```
 
-## R, IRkernel and RStudio
-We will be using R, another programming language, a lot in the program. We will use R both in Jupyter notebooks and in RStudio. To have R work in Jupyter notebooks we will also have to install the IR kernel.
+To test that your JupyerLab installation is functional, you can type `jupyter lab` into a terminal, which should open a new tab in your default browser with the JupyterLab interface.
 
-#### R
+## R, IRkernel and RStudio
 
-To obtain the latest R 4.0 packages, type the following into the terminal:
+R is another programming language that we will be using a lot in the MDS program. We will use R both in Jupyter notebooks and in RStudio.
 
-```
-sudo apt-add-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/'
-```
+#### R
 
-Next, add the key ID for the CRAN network:
+The version of R available in the default Ubuntu repositories (3.6) is older than the one we use in MDS (4.0). To obtain the latest R 4.0 packages, we need to add a new repository which is maintained directly by the r-project. To do this, first add the key for this repository by typing the following:
```

Comment:
We might want to use more clear language here 3.6.* and 4.0.*, respectively, or 4.0.0 or higher (more human readable) so that they don't fret about not matching on the patch version number.